data-pipeline/src/pipeline/input_schemas.py

```diff
@@ -262,146 +262,122 @@
-    "Other DfE/EFA Revenue Grants": "float",
-    "Other income - LA & other Government grants": "float",
-    "Government source, non-grant": "float",
-    "Academies": "float",
-    "Non-Government": "float",
-    "All income from facilities and services": "float",
-    "Income from catering": "float",
-    "Receipts from supply teacher insurance claims": "float",
-    "Donations and/or voluntary funds": "float",
-    "Other self-generated income": "float",
-    "Investment income": "float",
-    "Teaching staff": "float",
-    "Supply teaching staff": "float",
-    "Education support staff": "float",
-    "Administrative and clerical staff": "float",
-    "Premises staff": "float",
-    "Catering staff": "float",
-    "Other staff": "float",
-    "Indirect employee expenses": "float",
-    "Staff development and training": "float",
-    "Staff-related insurance": "float",
-    "Supply teacher insurance": "float",
-    "Building and Grounds maintenance and improvement": "float",
-    "Cleaning and caretaking": "float",
-    "Water and sewerage": "float",
-    "Energy": "float",
-    "Rent and Rates": "float",
-    "Other occupation costs": "float",
-    "Special facilities": "float",
-    "Learning resources (not ICT equipment)": "float",
-    "ICT learning resources": "float",
-    "Examination fees": "float",
-    "Educational Consultancy": "float",
-    "Administrative supplies - non educational": "float",
-    "Agency supply teaching staff": "float",
-    "Catering supplies": "float",
-    "Other insurance premiums": "float",
-    "Legal & Professionalservices": "float",
-    "Auditor costs": "float",
-    "Interest charges for Loan and Bank": "float",
-    "Direct revenue financing - Revenue contributions to capital": "float",
-    "PFI Charges": "float",
-    "Revenue reserve": "float",
-    "Total Grant Funding": "float",
-    "Direct Grants": "float",
-    "Community Grants": "float",
-    "Targeted Grants": "float",
-    "Total Self Generated Funding": "float",
-    "Total Income": "float",
-    "Supply Staff Costs": "float",
-    "Other Staff Costs": "float",
-    "Total Staff Costs": "float",
-    "Maintenance & Improvement Costs": "float",
-    "Premises Costs": "float",
-    "Catering Expenses": "float",
-    "Occupation Costs": "float",
-    "Total Costs of Supplies and Services": "float",
-    "Total Costs of Educational Supplies": "float",
-    "Costs of Brought in Professional Services": "float",
-    "Total Expenditure": "float",
-    "London Weighting": "string",
-    "Date joined or opened if in period": "string",
-    "Date left or closed if in period": "string",
+    "ACADEMYUPIN": "Int64",
+    "Company_Number": "string",
+    "Date joined or opened if in period:": "string",
+    "Date left or closed if in period:": "string",
+    "BNCH11110T (EFA Revenue Grants)": "float",
+    "BNCH11131 (DfE Family Revenue Grants)": "float",
+    "BNCH11141 (SEN)": "float",
+    "BNCH11142 (Other Revenue)": "float",
```

Comment:
What is the difference between this and `BNCH11160T (Other Revenue Income)`?

Reply:
Don't know what the T prefix stands for. I was just refering to the cost mapping sheet. This is one for Alan and Assad